src/index.js

```diff
@@ -2,16 +2,17 @@ import React from 'react';
 import ReactDOM from 'react-dom';
 import './index.css';
 import App from './App';
+import Parent from './ParentComponent';
 import reportWebVitals from './reportWebVitals';
 
 ReactDOM.render(
+  <React.Fragment>
   <React.StrictMode>
     <App />
-  </React.StrictMode>,
+    <Parent/>
+  </React.StrictMode>
```

Comment:
StrictMode you can add in App component itself

Reply:
Changes are done.